compose/backend.yml

```diff
@@ -173,19 +173,20 @@ services:
       - idam-database:/var/lib/postgresql/data
 
   service-auth-provider-api:
-    image: docker.artifactory.reform.hmcts.net/auth/service-auth-provider-api:47e18b53aad48ae8124744041988565e07dfed50
+    image: hmcts/service-auth-provider-app
     healthcheck:
       interval: 10s
       timeout: 10s
       retries: 10
     ports:
       - 4502:8080
```

Comment:
Shouldn't this be 8489 looking at
https://store.docker.com/community/images/hmcts/service-auth-provider-app/dockerfile ?

Reply:
Overriden to 8080 by `SERVER_PORT: 8080`. This is so that other containers using it don't have to be updated.